docs/docker-images/all-in-one.md

```diff
@@ -0,0 +1,76 @@
+This image contains `earthly`, `buildkit`, and some extra configuration to enable the two to work together. All that's missing is your source code! This image is mainly intended for use in containerized CI scenarios, or where maintaining a persistent installation of `earthly` isn't possible.
+
+## Quickstart
+
+Want to just get started? Here are a couple sample `docker run` commands that cover the most common use-cases:
+
+### Simple Usage
+
+```bash
+$ docker run --privileged -t -e NO_DOCKER=1 -v $(pwd):/workspace earthly/earthly:latest +for-linux
```

Comment:
Mind removing the `$`? We've not used `$` in other snippets, to allow the user to copy-paste easier.

Reply:
Preventing blind copy paste was why I did it ;) I can remove it if you would like. These CLIs will almost always require tweaking.